Cargo.toml

```diff
@@ -14,11 +14,11 @@ spec = "config_spec.toml"
 
 [dependencies]
 async-trait = "0.1.66"
-bitcoin = { version = "0.29.2", features = ["rand"] }
+bitcoin = { version = "0.30.2", features = ["rand"] }
 clap = { version = "4.4.6", features = ["derive"] }
 futures = "0.3.26"
 home = "0.5.5"
-lightning = { git = "https://github.com/orbitalturtle/rust-lightning", branch = "v0.0.118-custom", features = ["max_level_trace"] }
+lightning = { version = "0.0.120", features = ["max_level_trace"] }
```

Comment:
likewise, would be nice to have detail in the commit message outlining why we're bumping here